gateway/clientexec/clientexec.go

```diff
@@ -187,6 +196,17 @@ func (c *clientExec) run(inputPayload []byte, openSessionSpec map[string][]byte)
 		}
 		switch pkt.Type {
 		case pbclient.SessionOpenWaitingApproval:
+			session, err := sessionstorage.FindOne(c, c.sessionID)
+			if err != nil {
+				log.Error("failed obtaining session, err=%v", err)
+			}
+
+			issueInfo := jira.UpdateReviewJiraIssueTemplate{
+				ApiURL:    os.Getenv("API_URL"),
+				SessionID: c.sessionID,
+			}
+			jira.AddReviewCreatedJiraIssue(c.orgID, session.JiraIssue, issueInfo)
```

Comment:
I see these are called independently if the org has Jira integration configured or not. If they don't have it, how does this behave? I didn't find anything explicit at the Jira files that verifies and silently fails or something. Is it implicit somehow? How?

Reply:
Everything related to the Jira API goes through there in jira/client.go